Tasks/AzureCloudPowerShellDeployment/task.json

```diff
@@ -115,6 +115,20 @@
             "required": true
         },
         {
+            "name": "StorageAccountKeys",
+            "type": "multiLine",
+            "label": "Storage Account Keys",
+            "defaultValue": "",
+            "required": false,
+            "helpMarkDown": "Define storage keys corresponding to storage accounts being defined for diagnosticsExtensions.Format is WebRole1:Key1 WorkerRole1:Key2",
```

Comment:
Rephrase it. "Provide storage keys for diagnostics in Role:Storagekey format. The storage account for the roles mentioned will be picked from diagnostics config file. If there are no matching storage accounts in the config file, then the key will be ignored.